djangocms_blog/models.py

```diff
@@ -232,7 +232,7 @@ def get_title(self):
         return title.strip()
 
     def get_keywords(self):
-        return self.safe_translation_getter('meta_keywords').strip().split(',')
+        return self.safe_translation_getter('meta_keywords', default='').strip().split(',')
```

Comment:
@fmarco Yup, it's django-parler's fault. Filed an [issue](https://github.com/edoburu/django-parler/issues/121). But for now it's better to have ```default=''``` to make ```safe_translation_getter``` really safe :)